kits/beta-base/scripts/deploy_kit.js

```diff
@@ -40,36 +46,52 @@ module.exports = async (
     return await apm.newRepoWithVersion(name, acc, [1, 0, 0], c.address, '0x1245')
   }
 
-  log('owner', owner)
-
-  if (network == 'rpc' && false) { // TODO!!
-    log("Local testing network, exiting...")
-    return;
-  }
-
   let indexFileName
-  if (network != 'rpc' && network != 'devnet') {
-    indexFileName = 'index.js'
-  } else {
-    indexFileName = 'index_local.js'
+  if (!returnKit) {
```

Comment:
If `returnKit` is false and no ENS address is provided by param or in `ENV`, `indexFileName` below in line 58 will be undefined.

Reply:
That's true, adding a explicit error.